packages/shared/lib/services/plans/definitions.ts

```diff
@@ -0,0 +1,107 @@
+import type { PlanDefinition } from '@nangohq/types';
+
+export const plansList: PlanDefinition[] = [
+    {
+        code: 'free',
+        title: 'Free',
+        description: 'API authorization only.',
+        canUpgrade: true,
+        canDowngrade: false,
+        flags: {
```

Comment:
I would have called this attribute `features` but up to you

Reply:
yeah I hesitated, the fact that some are configurations of features and not just boolean made me change to flags